lib/r10k/action/puppetfile/install.rb

```diff
@@ -26,18 +26,18 @@ def visit_puppetfile(pf)
         end
 
         def visit_module(mod)
-          @update_force    = @update_force || false
+          @force    = @force || false
```

Comment:
I think this can be just `@force ||= false`? When false, this will be re-evaluated every time, but that's probably okay.

Reply:
The usage is similar to the `initialize` method in puppetfile.rb, but in this case the left and right hand sides reference the same variable, while in puppetfile.rb the right hand side refers to a parameter that might be nil while the left hand side refers to an instance variable.

Reply:
Changed.